# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/PathSkia.h
# Commit: 20b46afb225f
# Full Hash: 20b46afb225f7b0076e3fc0c000ce609ac79e0ba
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2023-04-11 09:27:51
# Regressor Bug: 1821512
# File Overlap Count: 1
# Description:
#   Bug 1821512 - Fix Skia usage for API changes. r=jrmuizel
#   
#   This just tries to address fairly random changes in the Skia API and correct
#   our usage of it in Moz2D and some other places.
#   
# ==============================================================================

diff -r b531f604ea4c -r 20b46afb225f gfx/2d/PathSkia.h
--- a/gfx/2d/PathSkia.h	Tue Apr 11 05:00:04 2023 +0000
+++ b/gfx/2d/PathSkia.h	Tue Apr 11 05:00:05 2023 +0000
@@ -91,6 +91,10 @@
 
   Maybe<Rect> AsRect() const override;
 
+  bool GetFillPath(const StrokeOptions& aStrokeOptions,
+                   const Matrix& aTransform, SkPath& aFillPath,
+                   const Maybe<Rect>& aClipRect = Nothing()) const;
+
  private:
   friend class DrawTargetSkia;
 